daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/TransactionVersion.scala

```diff
@@ -29,14 +29,21 @@ object TransactionVersion {
   private[this] val stringMapping = Values.iterator.map(v => v.protoValue -> v).toMap
 
   def fromString(vs: String): Either[String, TransactionVersion] =
-    stringMapping.get(vs).toRight(s"Unsupported transaction version $vs")
+    stringMapping.get(vs) match {
+      case Some(value) => Right(value)
+      case None =>
+        Left(s"Unsupported transaction version '$vs'")
+    }
 
   def assertFromString(vs: String): TransactionVersion =
     data.assertRight(fromString(vs))
 
-  val minVersion = Values.min
-  private[transaction] val minChoiceObservers = VDev
-  private[transaction] val minNodeVersion = VDev
+  val minVersion: TransactionVersion = Values.min
```

Comment:
Shouldn't this line somehow change given that we abandon value versions?

Reply:
I am not sure what you mean. 

note that `Values` has nothing to do with LF values, it is the sequences of all possible values the type `TransactionVersion`. Should I use a different name ?

Reply:
I think I've seen this called `All` in other places. Maybe that?